packages/manager/modules/billing/src/autoRenew/autorenew.constants.js

```diff
@@ -73,7 +73,22 @@ export const SERVICE_STATUS = {
 
 export const URL_PARAMETER_SEPARATOR = '%20';
 
+export const AUTORENEW_2016 = [
+  'ES',
+  'DE',
+  'IT',
+  'PL',
+  'PT',
+  'NL',
+  'GB',
+  'IE',
+  'SN',
+  'TN',
+  'MA',
+];
```

Comment:
**nitpick:** Classic use-case of `feature-flipping`. Why cant we use feature-flipping here?